### ISSUE TITLE
 Unit conversion

### ISSUE THREAD

**Issue by SlippyJimmy** [time4tea/gopro-dashboard-overlay#153]
I want to generate a running video overlay.but the cadence units is RPM,how to  convert to SPM(steps per mins)?the speed units is mph or kph,how to convert to PACE(**mins per km)?

**Comment by time4tea** [comment]
Hiya,

If the cadence values are correct, and its just the labelling that is wrong, you could perhaps use your own layout file?

The built-in layouts can be used as a base, to create any other layout you like.. for example, using the file: https://github.com/time4tea/gopro-dashboard-overlay/blob/main/gopro_overlay/layouts/default-1920x1080.xml

You could edit https://github.com/time4tea/gopro-dashboard-overlay/blob/main/gopro_overlay/layouts/default-1920x1080.xml#L47

to say something else?

There is some documentation on thexml layout format at: https://github.com/time4tea/gopro-dashboard-overlay/tree/main/docs/xml

For unit conversions of pace.. this is something that hasn't come up previously. I'll look to add it.

**Comment by SlippyJimmy** [comment]
First of all, thank you for your answer. Adding pace unit conversion is exciting for runners. See the picture below for the settings of SPM（steps per mins). I rechecked my fit file, and the value of cadence  is  PRM(Revolutions per mins) (as we all know,RPM is half of SPM). What should I do to double the value display?
<img width="809" alt="image" src="https://github.com/time4tea/gopro-dashboard-overlay/assets/45062658/f66ef38f-87ae-4f28-be17-798bcd9eace2">

**Comment by time4tea** [comment]
This is released in v0.107.0 - should be on PyPI shortly. Note that pace may be off by 1 spm, this will be fixed in the next release.

**Comment by time4tea** [comment]
You'll need to change the unit of the cadence metric to "spm"
```
<component type="metric" x="-140" y="36" metric="cadence" units="spm" dp="0" size="64" align="right" />
```
